test/runtests.jl

```diff
@@ -12,46 +12,58 @@ using FilePathsBase.TestPaths
 using UUIDs: uuid4
 using JSON3
 
+const AWSS3_TESTSETS = split(get(ENV, "AWSS3_TESTSETS", "MinIO,S3"), ',')
```

Comment:
I feel like this should be documented somewhere?

Reply:
I added something to the README